packages/server/dataloader/atlassianLoaders.ts

```diff
@@ -171,8 +167,42 @@ export const jiraIssue = (
                 })
               })
             )
+
+            const VALID_TYPES = ['string', 'number']
+            const INVALID_WORDS = ['color', 'name', 'description', 'environment']
+            const INVALID_NOT_SIMPLIFIED_FIELD = 'story point estimate'
+            const INVALID_SIMPLIFIED_FIELD = 'story points'
+            const simplified = !!issueRes.fields.project?.simplified
+
+            const possibleEstimationFieldNames = Array.from(
+              new Set(
+                Object.entries<{type: string}>(issueRes.schema)
+                  .filter(([fieldId, fieldSchema]) => {
+                    if (!VALID_TYPES.includes(fieldSchema.type)) return false
+                    const fieldName = issueRes.names[fieldId].toLowerCase()
+                    for (let i = 0; i < INVALID_WORDS.length; i++) {
+                      if (fieldName.includes(INVALID_WORDS[i]!)) return false
+                    }
+
+                    if (
+                      (!simplified && fieldName === INVALID_NOT_SIMPLIFIED_FIELD) ||
+                      (simplified && fieldName === INVALID_SIMPLIFIED_FIELD)
+                    ) {
+                      return false
+                    }
+
+                    return true
+                  })
```

Comment:
this filter fn seems like it may be big enough to be a named fn and passed in rather than inlined as an anonymous callback